### PR TITLE
Fix building for Java 11

### PR DESCRIPTION
**Pull request by ThadHouse**
Requires prerelease gradle 5.0

Also disables errorprone for now.

### CODE REVIEW

**Comment by JLLeitschuh** [comment]
Why Gradle 5.0-milestone-1? Not a criticism, more curiosity.

**Comment by ThadHouse** [comment]
I think Lompoc on Gradle 4.x was failing but it worked on 5.0 for some reason

File: src/main/java/robotbuilder/graph/CommandCellEditorDialog.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public CommandCellEditorDialog(CommandGraph graph, CommandCell command) {
 
     public void save() {
         // Save the parameters
-        Vector<Vector<Object>> dataVector = parameterTable.getModel().getDataVector();
+        Vector<Vector> dataVector = parameterTable.getModel().getDataVector();
```
Comment (JLLeitschuh) [inline src/main/java/robotbuilder/graph/CommandCellEditorDialog.java]: Erm, no, you should use `Vector<Vector<*>>` if you need to.

Or, if you want the compiler to infer the type with JDK 11 just use `val` for the type.

File: src/main/java/robotbuilder/ParameterSetsTable.java [hunk 2]
```diff
@@ -303,7 +303,7 @@ private boolean areParametersValid(int rowNum) {
         }
 
         @Override
-        public Vector<Vector<Object>> getDataVector() {
+        public Vector<Vector> getDataVector() {
```
Comment (JLLeitschuh) [inline src/main/java/robotbuilder/ParameterSetsTable.java]: You shouldn't drop the generic on the second vector, just return `Vector<Vector<*>>` or `Vector<Vector<? extends Object>>` might work as well.

Comment (ThadHouse) [reply src/main/java/robotbuilder/ParameterSetsTable.java]: I get a type not compatible error on Java 11 adding anything to the inner vector parameter.

File: src/main/java/robotbuilder/graph/CommandCellEditorDialog.java (hunk 1 above)
Comment (ThadHouse) [reply src/main/java/robotbuilder/graph/CommandCellEditorDialog.java]: `parameterTable.getModel().getDataVector();` actually returns a Vector of Vector. The internal vector has no return type in Java 11. And I would like to keep Java 8 building I think, so no `var`.

File: src/main/java/robotbuilder/ParameterSetsTable.java (hunk 2 above)
Comment (SamCarlberg) [reply src/main/java/robotbuilder/ParameterSetsTable.java]: @JLLeitschuh Java 9 changed the signature of the base method from returning a raw vector to a typed vector. See #107 for details

**Comment by ThadHouse** [comment]
This PR Closes #107

**Pull request merged**